### PR TITLE
[Enhance] Add 3dssd checkpoint

### PR DESCRIPTION
**Pull request by xiliu8006**
Add 3DSSD checkpoint and log download link

### CODE REVIEW

**Review by Tai-Wang — APPROVED** [review]

File: configs/3dssd/README.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ Some settings in our implementation are different from the [official implementat
...
+[1]: We report two different 3D object detection performance here. 78.11mAP is evaluated by our evaluation code and 79.17mAP is evaluated by the official development kit （so as that used in the paper and official code of 3DSSD ）. We found that the commonly used Python implementation of [`rotate_iou`](https://github.com/traveller59/second.pytorch/blob/e42e4a0e17262ab7d180ee96a0a36427f2c20a44/second/core/non_max_suppression/nms_gpu.py#L605) which is used in our KITTI dataset evaluation, is different from the official implemention in [KITTI benchmark](http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=3d).
```
Comment (ZwwWayne) [inline configs/3dssd/README.md:32]: Performance dropped. Need further check.

**Comment by ZwwWayne** [comment]
ping

**Comment by xiliu8006** [comment]
> ping

I meet some troubles, I am not able to run the 3DSSD evaluation code, because this official code is not able be compiled in centos7

File: configs/3dssd/README.md (hunk 1 above)
Comment (ZwwWayne) [inline configs/3dssd/README.md:32]: Update the model link and performance.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**